tests/integration/test_prusa_link.py

```diff
@@ -67,6 +69,91 @@ async def test_auth(printer: Printer, wui_client: aiohttp.ClientSession):
         await response.json()  # Tries to parse and throws if fails decoding
 
 
+async def test_status_responses(printer_factory, printer_flash_dir, data_dir):
```

Comment:
Generally, the goal is to make test cases as small as possible. So that when one part (test case) fails, the others are still gonna be tested.

Consider splitting this into several test cases (eg `test_api_version_endpoint`, `test_api_printer_endpoint`, etc).